omero/developers/Model/Units.rst

```diff
@@ -13,13 +13,13 @@ converting their internal unit to the OME default.
 Supported units
 ---------------
 
-* :javadoc:`Electric potential <omero/model/ElectricPotentialI.html>`
-* :javadoc:`Frequency <omero/model/FrequencyI.html>`
-* :javadoc:`Length <omero/model/LengthI.html>`
-* :javadoc:`Power <omero/model/PowerI.html>`
-* :javadoc:`Pressure <omero/model/PressureI.html>`
-* :javadoc:`Temperature <omero/model/TemperatureI.html>`
-* :javadoc:`Time <omero/model/TimeI.html>`
+* :javadoc_model:`Electric potential <omero/model/ElectricPotentialI.html>`
```

Comment:
Probably needs to be Blitz. To get this PR in might should first have a CI job that actually checks links.